ParabolInc/parabol
# feat(prompt-to-join): User is requesting access to your org notification

Fixes #7797

<img width="627" alt="image" src="https://user-images.githubusercontent.com/466991/231826299-0279a8ef-44f8-4367-98d3-bb8ca62826da.png">

https://lh3.googleusercontent.com/a/AGNmyxYt9RLKx4Lqa3USRNZ8Kw_oUXuf0mw4DlbHwXOR=s96-c


How to test:

- Follow testing guide from here #8003 to create matched org and prompt to join notification
- Click on `Request to join`
- See that team leads in matching org received notification from the screenshots above
